### PR TITLE
Update routing.md

### PR DESCRIPTION
**Pull request by takeitforwardsolutions**
Converted to English

##### Checklist

- [ ] `npm test` passes
- [ ] tests are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

##### Description of change

- any feature?
- close https://github.com/umijs/umi/ISSUE_URL

-----
[View rendered docs/docs/routing.md](https://github.com/takeitforwardsolutions/umi/blob/patch-1/docs/docs/routing.md)

### CODE REVIEW

**Comment by takeitforwardsolutions** [comment]
Converted to english

File: docs/docs/routing.md [hunk 1]
```diff
@@ -215,16 +215,16 @@ export default function(props) {
...
 
-## 传递参数给子路由
+## Passing parameters to subroutines
 
-通过 cloneElement，一次就好（Umi 2 时需要两次）。
+With cloneElement, it's fine once (Umi 2 requires twice).
 
 ```js
-import React from 'react';
+import React from'react';
```
Comment (ycjcl868) [inline docs/docs/routing.md:223]:
```suggestion
import React from 'react';
```

**Comment by neekey** [comment]
there seem to be multiple PRs about translating the `routing.md` suspending atm... 

- #4906 
- #4149 
- #4055 

@sorrycc what's the planing about dealing with them? or should UMI make a push to one of them to update (as they all have conflicts with the master). The issue with so many existing unmerged translation PRs for the same page is it stops people making contributes to it. (As I was going to translate the routing.md and then found out there were four PRs already)

**Comment by takeitforwardsolutions** [comment]
Thank You, please let me know; I could make myself available to translate and potentially some of my interns. Really enjoy the UMI project and would to expose it to a larger audience with translated material. Don Sent from Mail for Windows 10 From: NeekeySent: Tuesday, August 18, 2020 10:16 AMTo: umijs/umiCc: Donald Osgood @takeitforwardSolutions; AuthorSubject: Re: [umijs/umi] Update routing.md (#4989) there seem to be multiple PRs about translating the routing.md suspending atm...#4906#4149#4055@sorrycc what's the planing about dealing with them? or should UMI make a push to one of them to update (as they all have conflicts with the master). The issue with so many existing unmerged translation PRs for the same page is it stops people making contributes to it. (As I was going to translate the routing.md and then found out there were four PRs already)—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

**Pull request closed without merging**